packages/block-library/src/navigation/index.php

```diff
@@ -1530,13 +1542,17 @@ function block_core_navigation_insert_hooked_blocks_into_rest_response( $respons
 	return $response;
 }
 
-// Before adding our filter, we verify if it's already added in Core.
-// However, during the build process, Gutenberg automatically prefixes our functions with "gutenberg_".
-// Therefore, we concatenate the Core's function name to circumvent this prefix for our check.
+/*
+ *  Before adding our filter, we verify if it's already added in Core.
+ * However, during the build process, Gutenberg automatically prefixes our functions with "gutenberg_".
+ * Therefore, we concatenate the Core's function name to circumvent this prefix for our check.
+ */
 $rest_prepare_wp_navigation_core_callback = 'block_core_navigation_' . 'insert_hooked_blocks_into_rest_response';
 
-// Injection of hooked blocks into the Navigation block relies on some functions present in WP >= 6.5
-// that are not present in Gutenberg's WP 6.5 compatibility layer.
+/*
+ * Injection of hooked blocks into the Navigation block relies on some functions present in WP >= 6.5
+ * that are not present in Gutenberg's WP 6.5 compatibility layer.
+ */
```

Comment:
So all these comments are only really relevant for GB, not core, right? 'cause obviously when merging this change into core, these functions exist.

Reply:
The file is maintained in Gutenberg and those comments are relevant. Whilst I appreciate they aren't relevant for Core, we don't have a process for handling exclude comments at this time.

How critical is it to resolve this for RC 2?

Can we discuss a solution for RC 3?

Reply:
Just a general remark, not a blocker. I suppose we could remove remove these hardening checks & comments in the future though, e.g. in 6.6?

Reply:
I raised [this Discussion](https://github.com/WordPress/gutenberg/discussions/59786) in the hope we can improve the whole process for further releases.